django/repository/views.py

```diff
@@ -24,6 +24,7 @@ def get_queryset(self, *args, **kwargs):
             .filter(is_active=True)
             .prefetch_related("versions")
             .order_by("-date_updated")
+            .order_by("-pinned")
```

Comment:
You can (and should) use a single order by with two criteria, so
`.order_by("-pinned", "-date_updated")` should work if I remember correctly